packages/app/src/cli/services/function/build.ts

```diff
@@ -242,6 +242,18 @@ export async function runWasmOpt(modulePath: string) {
   await exec(command, args, {cwd: wasmOptDir})
 }
 
+export async function runTrampoline(modulePath: string) {
+  const trampoline = trampolineBinary()
+  await downloadBinary(trampoline)
+
+  const command = trampoline.path
+
+  const args = ['-i', modulePath, '-o', modulePath]
+  outputDebug(`command: ${command}`)
+  outputDebug('Applying trampoline to the wasm binary.')
```

Comment:
Nit: These could be combined into a single debug output. It would probably also be useful debugging output to include the arguments here too.